### PR TITLE
Implemet fix for PLT-2606 - Allow sending of invites from "Send Email Invite" with empty rows

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Allow sending of invites from "Send Email Invite" with empty rows

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/PLT-2606
GitHub: https://github.com/mattermost/platform/issues/6203

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks! Looks like there's a build failure @cpanato

**Comment by cpanato** [comment]
this is what happen when push at 1am :p @jasonblais

**Comment by lindalumitchell** [comment]
@jasonblais @cpanato this looks great! Tested inviting one email address, with additional email field blank; tested inviting two email addresses, with additional email field blank. Tested with and without name fields populated. All tests sent the invite emails as expected, and signup worked as expected. No issues found.

File: webapp/components/invite_member_modal.jsx [hunk 1]
```diff
@@ -114,22 +114,36 @@ class InviteMemberModal extends React.Component {
...
+                invite.index = index;
+                invite.email = ReactDOM.findDOMNode(this.refs['email' + index]).value.trim();
+                invite.firstName = ReactDOM.findDOMNode(this.refs['first_name' + index]).value.trim();
+                invite.lastName = ReactDOM.findDOMNode(this.refs['last_name' + index]).value.trim();
+                if (invite.email !== '') {
+                    invites.push(invite);
+                }
+            }
+        } else {
+            var inviteSingle = {};
```
Comment (crspeller) [inline webapp/components/invite_member_modal.jsx]: What's the point of this? Why not just have the loop run once?

Comment (cpanato) [inline webapp/components/invite_member_modal.jsx]: done

**Pull request merged**